osal/freertos/osal_common.c

```diff
@@ -156,3 +156,23 @@ int osal_sigaddset(sigset_t *set, int signum)
 {
     /* FreeRtos implementation */
 }
+
+void *osal_malloc(size_t size)
+{
+    /* FreeRtos implementation */
```

Comment:
What is the plan to implement the definitions for FreeRTOS target?

Reply:
pvPortMalloc, later sl_malloc for efr32